chunkator/__init__.py

```diff
@@ -11,9 +11,12 @@ def chunkator(queryset, chunk_size):
     length. We're saving at least a ``count()`` query on QuerySets, or a
     CPU-and-RAM-consuming ``len(queryset)`` query.
     """
-    pk = 0
+    pk = None
     while True:
-        page = queryset.filter(pk__gt=pk).order_by('pk')[:chunk_size]
+        queryset = queryset.order_by('pk')
```

Comment:
are you sure ? I mean , in common case, the order is denied by the model itself..